### PR TITLE
UserScores

### PR DESCRIPTION
**Pull request by RomanPudashkin**
Screenshot_20200730_195848

### CODE REVIEW

File: mu4/userscores/view/recentscoresmodel.cpp [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    if (months < monthInYear) {
+        return qtrc("userscores", "%1 months ago").arg(months);
+    }
+
+    int years = currentDate.year() - creationDate.year();
+    if (years == 1) {
+        return qtrc("userscores", "1 year ago");
+    }
+
+    return qtrc("userscores", "%1 years ago").arg(years);
```
Comment (igorkorsukov) [inline mu4/userscores/view/recentscoresmodel.cpp:188]: I think better move this function to framework/global/dataformatter.h

File: mu4/userscores/view/recentscoresmodel.cpp [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+        obj[SCORE_PATH_KEY] = path;
+        obj[SCORE_THUMBNAIL_KEY] = meta.val.thumbnail;
+        obj[SCORE_TIME_SINCE_CREATION_KEY] = timeSinceCreation(meta.val.creationDate);
+        obj[SCORE_ADD_NEW_KEY] = false;
+
+        recentScores << obj;
+    }
+
+    QVariantMap obj;
+    obj[SCORE_TITLE_KEY] = qtrc("scores", "New Score");
```
Comment (igorkorsukov) [inline mu4/userscores/view/recentscoresmodel.cpp:128]: "scores" -> "userscores"

Comment (RomanPudashkin) [reply mu4/userscores/view/recentscoresmodel.cpp]: fixed

File: mu4/userscores/view/recentscoresmodel.cpp (hunk 1 above)
Comment (RomanPudashkin) [reply mu4/userscores/view/recentscoresmodel.cpp]: fixed

File: mu4/userscores/internal/userscoresconfiguration.cpp [hunk 3]
```diff
@@ -16,20 +16,20 @@
...
 
 using namespace mu;
 using namespace mu::framework;
-using namespace mu::scores;
+using namespace mu::userscores;
 
-static std::string module_name("scores");
+static std::string module_name("userscores");
 
 static const Settings::Key RECENT_LIST(module_name, "scores/recentList");
```
Comment (Eism) [inline mu4/userscores/internal/userscoresconfiguration.cpp:30]: userscores

Comment (RomanPudashkin) [reply mu4/userscores/internal/userscoresconfiguration.cpp]: fixed

File: mu4/userscores/qml/MuseScore/UserScores/UserScoresContent.qml [hunk 4]
```diff
@@ -0,0 +1,129 @@
+import QtQuick 2.7
+import QtQuick.Layouts 1.15
```
Comment (vpereverzev) [inline mu4/userscores/qml/MuseScore/UserScores/UserScoresContent.qml:2]: QT 5.9 has no QtQuick.Layouts 1.15

Comment (RomanPudashkin) [reply mu4/userscores/qml/MuseScore/UserScores/UserScoresContent.qml]: fixed

**Pull request merged**